inspektor/lint.py

```diff
@@ -19,6 +19,9 @@
 from pylint.lint import Run
 
 from .path import PathChecker
+from .utils import process
+
+_PYLINT_HELP_TEXT = process.run('pylint --help').stdout
```

Comment:
I just noticed this outputs unnecessary `Running...` message. Could you please silence it? Apart from that it looks and work good (as far as my old pylint without `--score` says)